testsuite/MDAnalysisTests/coordinates/test_rdkit.py

```diff
@@ -355,17 +371,17 @@ def test_cache(self):
         assert cache.misses == 3
         mol = ucc.atoms.convert_to("RDKIT")
         assert cached_func.cache_info().hits == 1
```

Comment:
Is it worth also reloading `uc` and checking that the cache hits doesn't increase? (ignore if overly pedantic)